src/strkey.js

```diff
@@ -6,10 +6,123 @@ import isNull from "lodash/isNull";
 import isString from "lodash/isString";
 
 const versionBytes = {
-  accountId: 0x30,
-  seed:      0x90
+  publicKey:  6 << 3, // G
+  seed:      18 << 3, // S
+  hashTx:    19 << 3, // T
+  hashX:     23 << 3  // X
 };
 
+/**
+ * StrKey is a helper class that allows encoding and decoding strkey.
+ */
+export class StrKey {
```

Comment:
Yeah, I was thinking for some time before changing this but I decided to change it to match [stellar-core naming](https://github.com/stellar/stellar-core/blob/next-db-and-proto-2/src/crypto/StrKey.h#L18-L21). I think that names you listed are easy to understand for end users but can mislead developers working on the code (ex. seed/secret key, "what the heck is account id?"). But I'm OK with changing this. @jedmccaleb?